data/meson.build

```diff
@@ -16,10 +16,10 @@ endif
 
 appstream_file = i18n.merge_file(
   input: 'dev.tchx84.Portfolio.appdata.xml.in',
```

Comment:
rename the file as well, please !

Reply:
Sorry I missed that, file is renamed now. Thanks.

Reply:
I also renamed it in po/POTFILES (correct filename)